inginious/frontend/plugins/register_students/pages/api/add_course_students_csv_file_api.py

```diff
@@ -0,0 +1,149 @@
+import os
+import web
+import re
+import hashlib
+import random
+
+from inginious.frontend.plugins.utils.admin_api import AdminApi
+from inginious.frontend.plugins.utils import get_mandatory_parameter
+
+_static_folder_path = os.path.join(os.path.dirname(__file__), "static")
+
+
+class AddCourseStudentsCsvFile(AdminApi):
+    def API_POST(self):
```

Comment:
A little long the method it [smells](https://refactoring.guru/smells/long-method) :nose:. This also needs documentation cause the name will never be descriptive.